uint8 target_joint_id
string target_joint_name
float64 current
---
